dashboard/src/components/UpgradeForm/UpgradeForm.tsx

```diff
@@ -81,6 +81,8 @@ function UpgradeForm({
 
   const { availablePackageDetail, versions, schema, values, pkgVersion } = selected;
 
+  const packageCluster = availablePackageDetail?.availablePackageRef?.context?.cluster;
```

Comment:
Since it is a child component depending upon the `AppUpgrade` view, I wonder whether we should get perform the changes in the view instead. 

Given that a `UpgradeForm` will always be rendered if the `app` is defined, why not reducing the number of attributes being passed to this component?

But, perhaps we can just add this simple change and later on, review all the namespace/cluster being passed through. (I've added a note in our project board for having a discussion about it)

Also, the tests are failing, I think (not the e2e, but the dashboard_test).

As usual, thanks a lot for contributing to Kubeapps!

Reply:
Ah okay interesting, yeah maybe putting it in the parent component would be best. I can give it another one over when I fix the tests.